raiden/transfer/mediated_transfer/initiator_manager.py

```diff
@@ -279,6 +283,7 @@ def handle_transferrefundcancelroute(
         and refund_transfer.lock.expiration == original_transfer.lock.expiration
     )
 
+    # Weird naming, what do others think?
     is_valid_refund = channel.refund_transfer_matches_received(refund_transfer, original_transfer)
```

Comment:
This might even be a bug, because the signature is 
```
def refund_transfer_matches_received(
    refund_transfer: LockedTransferSignedState, received_transfer: LockedTransferUnsignedState
) -> bool:
```
but `original_transfer` should be `LockedTransferSignedState`.